libpod/image/utils.go

```diff
@@ -20,7 +20,11 @@ import (
 // a match on name:tag
 func findImageInRepotags(search imageParts, images []*Image) (*storage.Image, error) {
 	_, searchName, searchSuspiciousTagValueForSearch := search.suspiciousRefNameTagValuesForSearch()
-	var results []*storage.Image
+	type Candidate struct {
+		name  string
+		image *Image
```

Comment:
(This can probably store a `*storage.Image` directly.)

Reply:
It needs to be Image so I can check IsReadOnly.